mscore/inspector/inspector_accidental.ui

```diff
@@ -84,7 +84,7 @@
         <string>Reset value</string>
        </property>
        <property name="accessibleName">
-        <string>Reset Small value</string>
+        <string>Reset 'Small' value</string>
```

Comment:
Prefer double quotes? Or backticks?
Those single ticks are in the en_US and de translations since quite long, no complaints so far